config/systemd/node-problem-detector-metric-only.service

```diff
@@ -10,7 +10,7 @@ ExecStart=/home/kubernetes/bin/node-problem-detector --v=2 --logtostderr --enabl
           --exporter.stackdriver=/home/kubernetes/node-problem-detector/config/exporter/stackdriver-exporter.json \
           --config.system-log-monitor=/home/kubernetes/node-problem-detector/config/kernel-monitor.json,/home/kubernetes/node-problem-detector/config/docker-monitor.json,/home/kubernetes/node-problem-detector/config/systemd-monitor.json \
           --config.custom-plugin-monitor=/home/kubernetes/node-problem-detector/config/kernel-monitor-counter.json,/home/kubernetes/node-problem-detector/config/systemd-monitor-counter.json \
```

Comment:
Do we need to include the gke system stats monitor config here in the default systemd service? Should we just call this network-system-stats-monitor.json instead?

Reply:
I plan to add some more metrics to this config (but yeah there's nothing gke-specific about them). I'll rename them as such.